### PR TITLE
Add browser support and permissions detection

### PR DESCRIPTION
**Pull request by niuyunsi**

### CODE REVIEW

File: src/components/Conference.tsx [hunk 1]
```diff
@@ -97,6 +99,20 @@ export class Conference extends React.Component<IConferenceProps, IConferenceSta
         this.leaveRoom()
     }
 
+    private checkBrowserSupport() {
```
Comment (asinner) [inline src/components/Conference.tsx]: return boolean for support

File: src/components/Conference.tsx [hunk 2]
```diff
@@ -65,15 +67,15 @@ export class Conference extends React.Component<IConferenceProps, IConferenceSta
             localStream: undefined,
             remoteStreams: {},
         }
-
+        this.checkBrowserSupport();
```
Comment (asinner) [inline src/components/Conference.tsx]: return early if the browser is not supported

Comment (niuyunsi) [reply src/components/Conference.tsx]: Done

File: src/components/Conference.tsx (hunk 1 above)
Comment (niuyunsi) [reply src/components/Conference.tsx]: Done

**Review by asinner — APPROVED** [review]

**Pull request merged**